### DISCUSSION TITLE
How do I to use a Pad transform with a constant value?

### DISCUSSION THREAD

**Discussion by vanossj** [Project-MONAI/MONAI#2459] [Q&A]
I'm looking at the various pad transforms that end up using `np.pad` such as `SpacialPad`. I can set the `mode="constant"` but how do I provide the constant value to use? `np.pad` has a `constant_values` param but I don't see that getting set in `SpacialPad`

**Comment by rijobro** [comment]
See this issue and the corresponding PR to enable this functionality: https://github.com/Project-MONAI/MONAI/issues/2460.
